src/components/Footer.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PureComponent from 'react-pure-render/component';
+
+const styles = {
+  footer: {
+    height: '36px',
+    position: 'absolute',
+    bottom: '0',
+    overflow: 'hidden',
+    width: '100%',
+    backgroundColor: 'rgba(0,0,0,0.05)',
+    fontFamily: 'Helvetica',
+    fontSize: '14px',
+    color: 'rgba(0,0,0,0.87)',
+    margin: '0px',
+    padding: '8px 5%',
+  },
+};
+
+export default class Footer extends PureComponent {
+  render() {
+    return (
+      <div style={styles.footer}>
+        <p>Copyright &copy; 2016 UW Blueprint</p>
```

Comment:
Do you need `p`? Does it help with styling? If it doesn't then you can just remove it, so that the text is a direct child of `div`. But if it helps with styling then it's fine.